otherlibs/ocamlc_loc/test/ocamlc_loc_tests.ml

```diff
@@ -135,6 +135,32 @@ let%expect_test "warning" =
     ; related = []
     } |}]
 
+(* FIXME: unused value warning isn't parsed correctly - the file excerpt isn't
+   extracted *)
+let%expect_test "unused value" =
+  let raw_error =
+    String.trim
+      {|
+File "test.ml", line 1, characters 4-7:
+1 | let foo = ()
+        ^^^
+Error (warning 32 [unused-value-declaration]): unused value foo.
+    |}
+  in
+  String.split_lines raw_error
+  |> String.concat ~sep:"\r\n" |> Ocamlc_loc.parse |> Test.print_errors;
```

Comment:
We don't need this concat/split dance. it was present in the previous test because i was testing the `\r` trimming.